### PR TITLE
nkf: add page

### PR DESCRIPTION
**Pull request by pr0xy-t**
- [x] The page (if new), does not already exist in the repo.
- [x] The page has 8 or fewer examples.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/nkf.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Stdout shift_jis code:
+
+`nkf -s {{path/to/file.txt}}`
+
+- Overwrite as utf-8 code:
+
+`nkf -w --overwrite {{path/to/file.txt}}`
+
+- Set new line code to LF and overwrite(unix type):
```
[the remark this answers is not in the document]
Comment (imaiproxy) [reply pages/common/nkf.md]: It becomes easy to see😃

File: pages/common/nkf.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# nkf
+
+> Convert charcter code and new line code.
```
[the remark this answers is not in the document]
Comment (imaiproxy) [reply pages/common/nkf.md]: I will be careful😔

File: pages/common/nkf.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# nkf
+
+> Convert character code and new line code.
```
Comment (agnivade) [inline pages/common/nkf.md]: Please add "Network kanji filter" as the main heading.

From the man page here https://linux.die.net/man/1/nkf, it seems that it converts input kanji code from one encoding to another. I think that is more clear. The current description does not mention what does it convert to.

File: pages/common/nkf.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+- Stdout utf-8 code:
```
Comment (agnivade) [inline pages/common/nkf.md]: Stdout is not a verb. Please change it to "Convert to UTF-8 encoding"

File: pages/common/nkf.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+`nkf -w {{path/to/file.txt}}`
+
+- Stdout shift_jis code:
```
Comment (agnivade) [inline pages/common/nkf.md]: same as above.

File: pages/common/nkf.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`nkf -s {{path/to/file.txt}}`
+
+- Overwrite as utf-8 code:
```
Comment (agnivade) [inline pages/common/nkf.md]: Convert to UTF-8 encoding and overwrite the file.

Also, I see --overwrite has a suffix value in the man page. Is that not true ?

File: pages/common/nkf.md (hunk 5 above)
Comment (agnivade) [reply pages/common/nkf.md]: Seems to be another encoding from https://linux.die.net/man/1/nkf. I would suggest the same format as above.

File: pages/common/nkf.md (hunk 6 above)
Comment (agnivade) [reply pages/common/nkf.md]: Possible. But it would be good to confirm this.

File: pages/common/nkf.md (hunk 5 above)
Comment (sbrl) [reply pages/common/nkf.md]: Done.

**Comment by zdroid** [comment]
I checked the manpage and it seems all examples are written correctly. `--overwrite` requires a suffix which is the file name in this case.

**Comment by agnivade** [comment]
If I am not mistaken, the overwrite suffix should mean what gets appended to the end of the filename that gets overwritten. The file name is obviously part of the command, but suffix is something that gets added at the end, if specified. But it's optional anyways, so I think it's fine.

**Pull request merged**